sdks/python/apache_beam/testing/benchmarks/nexmark/queries/query0.py

```diff
@@ -0,0 +1,78 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Nexmark Query 0: Pass through (send and receive auction events)..
+
+The Nexmark suite is a series of queries (streaming pipelines) performed
+on a simulation of auction events. This query is a pass through that
+simply parses the events generated by the launcher. It serves as a test
+to verify the infrastructure.
+"""
+from __future__ import absolute_import
+from __future__ import print_function
+
+import csv
+import logging
+
+import apache_beam as beam
+from apache_beam.metrics.metric import Metrics
+from apache_beam.testing.benchmarks.nexmark.models import nexmark_model
+
+
+class ParseEventFn(beam.DoFn):
```

Comment:
Consider moving this DoFn to a shared utility file in a future PR. You could also make it a PTransform to simplify its usage.

Reply:
I moved it to nexmark_util.py.